examples/digits_dann_lightn/main.ipynb

```diff
@@ -0,0 +1,365 @@
+    "set_seed(seed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92f9c271",
+   "metadata": {},
+   "source": [
+    "## Setup Model\n",
+    "\n",
+    "Here, we use the previously defined configuration and dataset to set up the model we will subsequently train."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "987bea6a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model, train_params = get_model(cfg, dataset, num_channels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7afdb064",
+   "metadata": {},
+   "source": [
+    "Output reposrts on data file use."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d948cfa7",
+   "metadata": {},
+   "source": [
+    "## Setup Logger\n",
+    "\n",
+    "A logger is used to store output generated during and after model training. This information can be used to assess the effectiveness of the training and to identify problems."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b556269",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger, results, checkpoint_callback, test_csv_file = setup_logger(\n",
+    "    train_params, cfg.OUTPUT.DIR, cfg.DAN.METHOD, seed\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea2ce15d",
+   "metadata": {},
+   "source": [
+    "## Setup Trainer\n",
+    "\n",
+    "A trainer object is used to determine and store model parameters. Here, one is configured with information on how a model should be trained, and what hardware will be used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5794cdd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainer = pl.Trainer(\n",
+    "    progress_bar_refresh_rate=cfg.OUTPUT.PB_FRESH,  # in steps\n",
+    "    min_epochs=cfg.SOLVER.MIN_EPOCHS,\n",
+    "    max_epochs=cfg.SOLVER.MAX_EPOCHS,\n",
+    "    callbacks=[checkpoint_callback],\n",
+    "    logger=False,\n",
+    "    gpus=gpus)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19f5a1bd",
+   "metadata": {},
+   "source": [
+    "Output reports on available GPU and TPU resources."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6e646cf",
+   "metadata": {},
+   "source": [
+    "## Train Model\n",
+    "\n",
+    "Optimise model parameters using the trainer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b082670",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainer.fit(model)\n",
+    "results.update(\n",
+    "    is_validation=True, method_name=cfg.DAN.METHOD, seed=seed, metric_values=trainer.callback_metrics,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c311a3ac",
+   "metadata": {},
+   "source": [
+    "## Test Optimised Model\n",
+    "\n",
+    "Check performance of model optmised with training data against test data which was not used in training."
```

Comment:
typo "optmised". Again a spell check plug-in may help.

Reply:
Thanks. I now have one installed.